tests/cypress/integration/reportRepair/findRepair.spec.js

```diff
@@ -0,0 +1,68 @@
+import {
+  navigateToFindRepairPage
+} from '../../support/helpers';
+
+function loadFindRepairPage() {
+  navigateToFindRepairPage();
+}
+
+describe('find repair', () => {
+  const repairDescriptionTextInputId = 'repair-description-text-input';
+
+  context('Content', () => {
+    before(() => {
+      loadFindRepairPage();
+    });
+    it('displays the title', () => {
+      cy.contains('Find your repair appointment');
+    });
+
+    it('displays what the repair number should include', () => {
+      cy.contains('Repair number');
+      cy.contains('Your repair number is in your confirmation email or text message');
+    });
+
+    it('displays postcode', () => {
+      cy.contains('Postcode');
+    });
+
+    it('Displays uknown repair number information box', () => {
+      cy.contains(`I don't know my repair number`).click()
+      cy.contains(`Contact us via telephone on ${Cypress.env('OUT_OF_HOURS_CUSTOMER_SERVICES_TELEPHONE_NUMBER')}`)
+      cy.contains(`Our call centre is open between ${Cypress.env('CUSTOMER_SERVICES_OPENING_HOURS_DESCRIPTION')}`)
+    });
+  });
+
+  context('Behaviour', () => {
+    context('Validation', () => {
+      beforeEach(() => {
+        loadFindRepairPage();
+      });
+      context('When a user doesn\'t type anything', () => {
```

Comment:
Add a test to verify error shown when either one of these fields are empty individually too, i.e. when postcode is entered and repair number is not and vice versa.

Reply:
Sorry, I didn't see them earlier.